src/vmm/src/default_syscalls/mod.rs

```diff
@@ -106,10 +108,13 @@ fn create_ioctl_seccomp_rule() -> Result<Vec<SeccompRule>, Error> {
     let mut rule = or![
         and![Cond::new(1, ArgLen::DWORD, Eq, KVM_RUN)?],
         and![Cond::new(1, ArgLen::DWORD, Eq, KVM_GET_DIRTY_LOG)?],
-        and![Cond::new(1, ArgLen::DWORD, Eq, FIONBIO)?],
-        // Triggered on microvm shutdown. If you remove it, tests will likely still pass,
-        // because the sigsys_handler exits the process.
+        // Triggered on shutdown, when Firecracker launched from a shell.
+        and![Cond::new(1, ArgLen::DWORD, Eq, TCGETS)?],
+        // Triggered on shutdown, when Firecracker launched from a shell.
+        and![Cond::new(1, ArgLen::DWORD, Eq, TCSETS)?],
+        // Triggered on shutdown.
         and![Cond::new(1, ArgLen::DWORD, Eq, TIOCGWINSZ)?],
```

Comment:
Instead of `Triggered on shutdown`, I'd be specific about the fact that these ioctls modify the terminal setting. Moreover, I'm curious whether Firecracker needs these ioctls at all if it runs without a serial console, which is the one that links it to stdin and stdout in the first place. I think it's worth trying out (under `strace`) a Firecracker build without the serial console, and without the terminal configurations, to check if these ioctls show up.

Reply:
thanks for the feedback. i tested without the serial console and indeed, these ioctls were not triggered. i will update the comments